src/style.css

```
@@ -0,0 +1,137 @@
+
+@font-face {
+  font-family: "Avenir Book";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: "Book";
+}
+
+@font-face {
+  font-family: "Avenir Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-weight: normal;
+}
+
+/*styling simple html tags  */
+* {
+  margin: 0 auto;
+  border: none;
+  padding: 0;
+  box-sizing: border-box;
+}
```

use * only for box-sizing
```suggestion
* {
  box-sizing: border-box;
}
```